HorizonEditor/src/EditorLayer.cpp

```diff
@@ -284,6 +284,12 @@ void EditorLayer::onRenderImgui()
 
 		if (ImGui::BeginMenu("View"))
 		{
+			if (ImGui::MenuItem("Components")) EditorData::s_ShowComponentsPanel = !EditorData::s_ShowComponentsPanel;
+			if (ImGui::MenuItem("Object Hierarchy")) EditorData::s_ShowObjectHierarchyPanel = !EditorData::s_ShowObjectHierarchyPanel;
+			if (ImGui::MenuItem("Viewport")) EditorData::s_ShowViewportPanel = !EditorData::s_ShowViewportPanel;
+			if (ImGui::MenuItem("Sprites")) EditorData::s_ShowSpritesPanel = !EditorData::s_ShowSpritesPanel;
+			if (ImGui::MenuItem("Content Browser")) EditorData::s_ShowContentBrowserPanel = !EditorData::s_ShowContentBrowserPanel;
```

Comment:
Great. But it would be slightly better if we could show some kind of indicator in the view menu, which shows what panels are hidden and which panels are visible. But I think this can be added anytime later as well, so no worries.

Reply:
Yeah, We can probably keep adding more to it later, now that we have the basic functionality up.